src/components/VideoCard/AddPlaylistBox/AddPlaylistBox.jsx

```diff
@@ -1,10 +1,46 @@
 import "./AddPlaylistBox.css";
-import { AiOutlineCloseCircle} from "../../../icons/icons";
+import { AiOutlineCloseCircle } from "../../../icons/icons";
+import { useVideoContext, useAuthContext } from "../../../context/context";
+import {useNavigate} from "react-router-dom" ; 
+import {
+  addPlaylistHandler,
+  addVideoToPlaylistCall,
+  removeVideoFromPlaylistCall,
+} from "../../../utils/utils";
+import { useInputHandler } from "../../../hooks/customHooks";
+import { isPresentInState } from "../../../utils/utilCalls";
 
-function AddToPlaylistBox({ setShowPlaylistMenu }) {
-  const newPlaylistHandler=(e)=>{
+function AddToPlaylistBox({ setShowPlaylistMenu, video }) {
+  const navigate = useNavigate();
+  const { userState } = useAuthContext();
+  const { videoStates, dispatch } = useVideoContext();
+  const { playlists } = videoStates;
+  const { inputState, inputUpdate } = useInputHandler({
+    title: "",
+    description: "",
+  });
+
+  const newPlaylistHandler = (e) => {
     e.preventDefault();
-  }
+    if (userState.id) {
+      if (playlists.some((item) => item.title === inputState.title)) {
+      } else {
+        addPlaylistHandler(inputState, userState?.id, dispatch);
```

Comment:
suggestion: you can clear out the input field after you create a new playlist.

Reply:
Made the change